app/handlers.go

```diff
@@ -0,0 +1,28 @@
+package app
+
+import (
+	"net/http"
+

```

Comment:
is this line left blank intentionally ? like, is it a good practice ?

Reply:
Yes, it is left intentionally.  This blank line was created by goimports tool automatically run by my IDE when I save a .go file.

It looks like a common practice to either use this tool or to group imports by hand into sections (e.g. I found the same practice in one of the first randomly opened gin source files - https://github.com/gin-gonic/gin/blob/master/render/json.go)

We could also separate our internal (starting with `"github.com/gujjsPet/image-transform-service"`) packages from 3rd party packages downloaded from github like gin and maybe some more in the future